### ISSUE TITLE
Wonder2

### ISSUE THREAD

**Issue by shanqiu24** [2gunsu/monocon-pytorch#4]
Hey guys，I come again，the pre-trained Backbone must work better than a zero-based one? I mean starting from scratch, the overfitting problem is unavoidable? And the best number of layers you've tried is 34, right? Have you ever tried 100+ DLA？

**Comment by 2gunsu** [comment]
Hello😊

1. We have used ImageNet pretrained weights on the DLA-34 backbone as in the original code.  I am also curious about the overfitting problem you mentioned, so I am training from scratch. However, my opinion is that there is a risk of overfitting because the KITTI 3D Object Detection dataset is not that large.
2. You're right. We trained and tested using only the DLA-34 backbone. I am training the model using pretrained DLA-102 backbone. This experiment will take longer. (This is because DLA-34 has 19 million parameters and DLA-102 has 49 million parameters.)

I will share the experimental results for both questions here.

**Comment by shanqiu24** [comment]
> 



> Hello😊
> 
> 1. We have used ImageNet pretrained weights on the DLA-34 backbone as in the original code.  I am also curious about the overfitting problem you mentioned, so I am training from scratch. However, my opinion is that there is a risk of overfitting because the KITTI 3D Object Detection dataset is not that large.
> 2. You're right. We trained and tested using only the DLA-34 backbone. I am training the model using pretrained DLA-102 backbone. This experiment will take longer. (This is because DLA-34 has 19 million parameters and DLA-102 has 49 million parameters.)
> 
> I will share the experimental results for both questions here.

Got it，I appreciate it~😊

**Comment by 2gunsu** [comment]
Thank you for waiting.   
The experimental results are out, please refer to the table below.  
For a fair comparison, the seed of all experiments was fixed to 0.  

Please understand that **DLA34 (Pretrained)** is not a reproduced result.
Because the best performance written in the README is the result obtained from a random seed.

### 3D Metric on Car Class
|               | AP40@Easy     | AP40@Mod.     | AP40@Hard     |
| ---------     | ---------     |-----------    |-----------    |
| Official      | 26.33         | 19.03         | 16.00         |
| DLA34 (Pretrained)      | 24.29         | 17.95         | 15.24         |
| DLA34 (Non-Pretrained)      | 22.71         | 16.72         | 14.00         |
| DLA102 (Pretrained)      | 25.33         | 18.23         | 15.41         |

#### Q1.
It can be seen that there is a performance gap between **DLA34 (Pretrained)** and **DLA34 (Non-Pretrained)**.  
So, unless you have sophisticated training tricks, it seems good to use pre-trained weights.

#### Q2.
Comparing **DLA34 (Pretrained)** and **DLA102 (Pretrained)**, the performance when using a heavier model is better, but the difference is not large. Therefore, in my opinion, using DLA-34 like other papers currently published would be the most appropriate considering the trade-off between training time and performance.

**Comment by shanqiu24** [comment]
> 



> Thank you for waiting. The experimental results are out, please refer to the table below. For a fair comparison, the seed of all experiments was fixed to 0.
> 
> Please understand that **DLA34 (Pretrained)** is not a reproduced result. Because the performance written in the README is the result obtained from a random seed.
> 
> ### 3D Metric on Car Class
> AP40@Easy	AP40@Mod.	AP40@Hard
> Official	26.33	19.03	16.00
> DLA34 (Pretrained)	24.29	17.95	15.24
> DLA34 (Non-Pretrained)	22.71	16.72	14.00
> DLA102 (Pretrained)	25.33	18.23	15.41
> #### Q1.
> It can be seen that there is a performance gap between **DLA34 (Pretrained)** and **DLA34 (Non-Pretrained)**. So, unless you have sophisticated training tricks, it seems good to use pre-trained weights.
> 
> #### Q2.
> Comparing **DLA34 (Pretrained)** and **DLA102 (Pretrained)**, the performance when using a heavier model is better, but the difference is not large. Therefore, in my opinion, using DLA-34 like other papers currently published would be the most appropriate considering the trade-off between training time and performance.

oh bro ,i got it ,thanks alot~😊
